operator/duties/attester.go

```diff
@@ -0,0 +1,270 @@
+package duties
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	eth2apiv1 "github.com/attestantio/go-eth2-client/api/v1"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/logging/fields"
+)
+
+type AttesterHandler struct {
+	baseHandler
+
+	duties            *Duties[phase0.Epoch, *eth2apiv1.AttesterDuty]
+	fetchCurrentEpoch bool
+	fetchNextEpoch    bool
+}
+
+func NewAttesterHandler() *AttesterHandler {
+	return &AttesterHandler{
+		duties: NewDuties[phase0.Epoch, *eth2apiv1.AttesterDuty](),
+	}
+}
+
+func (h *AttesterHandler) Name() string {
+	return spectypes.BNRoleAttester.String()
+}
+
+// HandleDuties manages the duty lifecycle, handling different cases:
+//
+// On First Run:
+//  1. Fetch duties for the current epoch.
+//  2. If necessary, fetch duties for the next epoch.
+//  3. Execute duties.
+//
+// On Re-org:
+//
+//	If the previous dependent root changed:
+//	    1. Fetch duties for the current epoch.
+//	    2. Execute duties.
+//	If the current dependent root changed:
+//	    1. Execute duties.
+//	    2. If necessary, fetch duties for the next epoch.
+//
+// On Indices Change:
+//  1. Execute duties.
+//  2. Reset duties for the current epoch.
+//  3. Fetch duties for the current epoch.
+//  4. If necessary, fetch duties for the next epoch.
+//
+// On Ticker event:
+//  1. Execute duties.
+//  2. If necessary, fetch duties for the next epoch.
+func (h *AttesterHandler) HandleDuties(ctx context.Context, logger *zap.Logger) {
+	logger = logger.With(zap.String("handler", h.Name()))
+	logger.Info("starting duty handler")
+
+	h.fetchCurrentEpoch = true
+	if h.shouldFetchNexEpoch(h.network.Beacon.EstimatedCurrentSlot()) {
+		h.fetchNextEpoch = true
+	}
+	h.fetchFirst = true
```

Comment:
this could be just 
```
if h.shouldFetchNexEpoch(h.network.Beacon.EstimatedCurrentSlot()) {
		h.fetchNextEpoch = true
	}
```
if you would set `h.fetchCurrentEpoch` and `h.fetchFirst` to true in `NewAttesterHandler`